content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,89 @@ description: Istio 1.5 release notes.
+- **Enabled** [automatic protocol detection](https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection-experimental) by default #18666.
+
+## Security
+
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into `PeerAuthentication` and `RequestAuthentication` respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** Authorization Policy supports deny action and exclusion semantics.
+- **Added** auto mutual TLS. This beta feature is enabled by default.
+- **Improved** node agent by merging with Pilot agent. This improves the security posture by removing the requirement of a pod security policy.
+- **Improved** Istio by including certificate provisioning functionality within Pilot.
+- **Improved** Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters that [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported.
+- **Improved** Istio's Pilot Agent by provisioning key and certificates for Prometheus consumption.
+- **Graduated** SDS which provides identity provisioning for Istio.
+
+## Telemetry
+
+- **Added** TCP protocol support for v2 telemetry.
+- **Added** gRPC response status code support in metrics/logs.
+- **Added** support for Istio Canonical Service.
+- **Improved** stability of v2 telemetry pipeline.
+- **Added** alpha-level support for configurability in v2 telemetry.
+- **Added** support for populating AWS platform metadata in Envoy node metadata.
+- **Improved** Stackdriver adapter for Mixer to support configurable flush intervals for tracing data.
+- **Added** support for a headless collector service to the Jaeger addon.
+- **Fixed** `kubernetesenv` adapter to provide proper support for pods that contain a dot in their name.
+- **Improved** the Fluentd adapter for Mixer to provide millisecond-resolution in exported timestamps.
+
+
+## Configuration management
+
+## Operator
+
+- **Improved** the alpha `IstioControlPlane` API by replacing with the `IstioOperator` API.
+**Added** `istioctl operator init` and `istioctl operator remove` commands istio/operator#643.
+**Improved** reconciliation speed with caching #20344.
+
+## `area/perf and scalability`
+
+- **Improved** Skip unused services while generate clusters for the gateways #20124.
+- **Improved** Skip calling `updateEDS` for headless service #18952.
+- disable `SNI-DNAT` at ingress gateway by default #18431.
+- The declaration of err overrides #19473.
+- create slice with capacity, when capacity is known #18504.
+
+## `area/test and release`
+
+- **Added** Publish a `docker/istioctl` image #19079.
+
+## `istioctl`
+- **Added** mTLS analyzer - #18350
```

Comment:
can this map to the concrete istioctl cmd?  is this the new istioctl authn replacement?